packages/sheriff-types/package.json

```diff
@@ -7,14 +7,12 @@
   "files": [
     "src"
   ],
-  "main": "./src/index.ts",
-  "types": "./src/index.ts",
   "module": "./src/index.ts",
+  "types": "./src/index.ts",
   "exports": {
     ".": {
       "types": "./src/index.ts",
-      "import": "./src/index.ts",
-      "require": "./src/index.ts"
```

Comment:
why are we removing require? Wouldn't this break CJS builds?

Reply:
Oh, is sheriff-types published? Oh, yes it is. As it wasn't bundled, I assumed it was internal only.

Reply:
Oh wait, you mean that there is no runtime code, so we can just export the `"types": "./src/index.ts"` and don't need the rest. I think you are right, I just copy/pasted that part of the config from the other packages. If this work as before, i'm fine with this change.

Reply:
The CJS/ESM distinction still matters with types.
However, the types won't be required or imported, they're only types.